src/generic/create-or-rerun-course/CreateOrRerunCourseForm.jsx

```diff
@@ -201,6 +201,15 @@ const CreateOrRerunCourseForm = ({
 
   return (
     <div className="create-or-rerun-course-form">
+      {errors[TOTAL_LENGTH_KEY] && (
+        <Form.Control.Feedback
+          className="feedback-error"
+          type="invalid"
+          hasIcon={false}
+        >
+          {errors[TOTAL_LENGTH_KEY]}
+        </Form.Control.Feedback>
+      )}
```

Comment:
To match the design of the pages, can you convert this into an `<Alert/>` that is not dismissable?

Reply:
Hi @KristinAoki ! 
I did. It looks better. Thanks for the tip! But I'm not quite sure if it's implemented well in the context of the code.

<img width="1384" alt="12345" src="https://github.com/user-attachments/assets/195f06e8-8b84-4a67-9a61-c36f9fe52aba">